docs/api/node.md

```diff
@@ -205,25 +209,30 @@ enum InterviewStage {
 	ProtocolInfo,
 	/** The node has been queried for supported and controlled command classes */
 	NodeInfo,
+
```

Comment:
This comes from removed single-line comments. Not pretty, but I see no simple way around it.